config/locales/simple_form.he.yml

```diff
@@ -5,9 +5,13 @@ he:
       defaults:
         avatar: 'PNG, GIF או JPG. מקסימום 2MB. גודל התמונה יוקטן ל-120x120px'
         display_name: 'מקסימום 30 תווים'
```

Comment:
Syntax error, can't be value + children at the same time, either one or the other